python/setup.py

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env python
+
+from setuptools import setup
+
+exec(compile(open("pyspark/pyspark_version.py").read(), 
+   "pyspark/pyspark_version.py", 'exec'))
+VERSION = __version__
+
+setup(name = 'pyspark',
+   version = VERSION,
+   description = 'Apache Spark Python API',
+   author = 'Prabin Banka',
+   author_email = 'prabin.banka@imaginea.com',
```

Comment:
If we want this in the Apache repo, then these individual author tags should be replaced with references to Apache.